stock_replenishment_mrp_bom_selection/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    def action_replenish(self):
+        # The wizard is only necessary if the production route is marked and the product
+        # has defined BOMs.
+        if (
+            self.route_id.rule_ids[:1].picking_type_id.code == "mrp_operation"
+            and self.product_id.bom_ids
+        ):
+            view_id = self.env.ref(
```

Comment:
AFAIK, there's no need to explicitly indicate the view here. Only with the model, the view will be selected automatically.